From 10cf1fb9c689e1fe6e9c4e822f754d68b3303659 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Thu, 2 Jul 2015 09:43:35 +0100
Subject: [PATCH 2/7] bootchart: drop log_info spam to serial console.

---
 src/bootchart.c | 2 --
 src/svg.c       | 2 --
 2 files changed, 4 deletions(-)

diff --git a/src/bootchart.c b/src/bootchart.c
index 0221167..c103812 100644
--- a/src/bootchart.c
+++ b/src/bootchart.c
@@ -497,8 +497,6 @@ int main(int argc, char *argv[]) {
                 return EXIT_FAILURE;
         }
 
-        log_info("systemd-bootchart wrote %s\n", output_file);
-
         r = do_journal_append(output_file);
         if (r < 0)
                 return EXIT_FAILURE;
diff --git a/src/svg.c b/src/svg.c
index e17fca5..f95b863 100644
--- a/src/svg.c
+++ b/src/svg.c
@@ -173,8 +173,6 @@ static int svg_title(FILE *of, const char *build, int pscount, double log_start,
                 xsprintf(filename, "/sys/block/%s/device/model", rootbdev);
 
                 r = read_one_line_file(filename, &model);
-                if (r < 0)
-                        log_info("Error reading disk model for %s: %m\n", rootbdev);
         }
 
         /* various utsname parameters */
-- 
2.13.0

