From 99b259498bc05dbbd438fc5e578200b4ef71b742 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Fri, 2 Jun 2017 13:37:08 -0700
Subject: [PATCH 5/7] Mount /proc early during boot

---
 src/bootchart.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/bootchart.c b/src/bootchart.c
index eb7b7ab..69c7e13 100644
--- a/src/bootchart.c
+++ b/src/bootchart.c
@@ -45,6 +45,8 @@
 #include <time.h>
 #include <unistd.h>
 
+#include <sys/mount.h>
+
 #ifdef HAVE_LIBSYSTEMD
 #include <systemd/sd-journal.h>
 #endif
@@ -326,6 +328,10 @@ int main(int argc, char *argv[]) {
         bool has_procfs = false;
 
         parse_conf();
+        
+        if (access("/proc/cmdline", R_OK) != 0) {
+                mount("/proc", "/proc", "proc", 0, NULL);
+        }
 
         r = parse_argv(argc, argv);
         if (r < 0)
-- 
2.13.0

